src/screens/Dashboard.js

```diff
@@ -191,7 +191,6 @@ const TokensHeader = () => (
 const styles = StyleSheet.create({
   wrapper: {
     flex: 1,
-    marginTop: 8,
```

Comment:
Should we keep the margin, but paint the background with the same grey color as the rest?

Reply:
Implemented this on [c7eabdb](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/674/commits/c7eabdbe35cb9cdb78829299f0e7f35eb07c63fb). Indeed it looks better.